src/patternfly/layouts/Gallery/docs/code.md

```diff
@@ -1,6 +1,6 @@
 ## Overview
 
-A Gallery layout makes all of the children a uniform size, the children are displayed horizontally, and they wrap as needed.
+The Gallery Layout is designed so that all of its children are of uniform size, displayed horizontally and wrap responsively.
```

Comment:
Changing to "...display horizontally, and wrap responsively." will make Oxford comma fans happy and keep the clauses parallel.